### PR TITLE
LPS-165647 Add property to enable Search by default, in SelectDDMFormField

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-165647

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:forms

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js [hunk 1]
```diff
@@ -475,6 +476,7 @@ const Main = ({
 	options = [],
 	predefinedValue = [],
 	readOnly = false,
+	defaultSearch = false,
```
Comment (alinedoleron) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:479]: Hi, @p-albuquerque. Just a simple adjustment. Please, could you sort the `defaultSearch` this property? It needs to be before the `editingLanguageId` prop.
Thanks! =)

**Review by alinedoleron — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js (hunk 1 above)
Comment (p-albuquerque) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js]: done.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java [hunk 2]
```diff
@@ -72,6 +73,8 @@ public Map<String, Object> getParameters(
 			GetterUtil.getBoolean(ddmFormField.getProperty("alphabeticalOrder"))
 		).put(
 			"dataSourceType", ddmFormField.getDataSourceType()
+		).put(
+			"defaultSearch", _hasDefaultSearch(ddmFormField)
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java:77]: Hey @p-albuquerque,
We could replace this method with this statement:
`GetterUtil.getBoolean(ddmFormField.getProperty("defaultSearch"))` 

What do you think?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi @p-albuquerque 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java (hunk 2 above)
Comment (p-albuquerque) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java:77]: i will see if this works, thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js (hunk 1 above)
Comment (alinedoleron) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js]: Thanks!! From FE side it's good!

**Review by alinedoleron — APPROVED** [review]
Front end is good!

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
Hey @p-albuquerque,
Did you run the unit/integration tests from the DDM side?

**Comment by p-albuquerque** [comment]
ci:relevant

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:reevaluate:1277401_5477

**Comment by carolmariaabb** [comment]
Hey @p-albuquerque,

If you haven't changed anything in your code since the last execution of `ci:test:relevant`, then feel free to manually forward this PR to Brian.

Thanks!

**Comment by p-albuquerque** [comment]
resent here: https://github.com/brianchandotcom/liferay-portal/pull/125044

**Comment by p-albuquerque** [comment]
ci:forward

**Pull request closed without merging**